protocol/messenger_activity_center.go

```diff
@@ -318,11 +399,126 @@ func (m *Messenger) handleActivityCenterAccepted(state *ReceivedMessageState, me
 }
 
 func (m *Messenger) handleActivityCenterDismissed(state *ReceivedMessageState, message protobuf.SyncActivityCenterDismissed) error {
-	resp, err := m.DismissActivityCenterNotifications(context.TODO(), toHexBytes(message.Ids), false)
+	resp, err := m.DismissActivityCenterNotifications(context.TODO(), toHexBytes(message.Ids), message.Clock, false)
 
 	if err != nil {
 		return err
 	}
 
 	return state.Response.Merge(resp)
 }
+
+func (m *Messenger) handleSyncActivityCenterNotificationState(state *ReceivedMessageState, a *protobuf.SyncActivityCenterNotificationState) error {
+	s := &ActivityCenterState{
+		HasSeen:   a.HasSeen,
+		UpdatedAt: a.UpdatedAt,
+	}
+	_, err := m.persistence.UpdateActivityCenterNotificationState(s)
```

Comment:
@qfrank if this handling is `handleSyncActivityCenterNotificationState`, so it's about singular, not plural, how come that we don't set anywhere any notification id? I mean how do we know which notification should be updated?

Reply:
there is an [API](https://github.com/status-im/status-go/blob/3bc07aac5a887595245393dd5626e9b726723c3c/services/ext/api.go#L1136) which mark all notifications as seen

Reply:
Ahh ok if it refers to all notifications, then ok (though the naming is a bit misleading).